### PR TITLE
Added functionality to anonymize domains  and general strings...

### PR DESCRIPTION
**Pull request by dameyerdave**
… in addition to ip addresses.

Added rules for fortigate, infoblox, exchange

Changed to python version 3

### CODE REVIEW

File: loganon [hunk 1]
```diff
@@ -1,30 +1,36 @@
-#!/usr/bin/env python
+#!/usr/local/bin/python3
```
Comment (sebix) [inline loganon]: ?!

Comment (dameyerdave) [reply loganon]: Sorry, was a quick change for my environment- --> I changed it to /usr/bin/env python3

**Comment by croessner** [comment]
Thank you very much. I will revie the changes soon 😊

**Comment by dameyerdave** [comment]
Thank you!

**Pull request merged**